### PR TITLE
Issue #13045: Node.js 12 actions deprecated

### PR DESCRIPTION
**Pull request by AayushSaini101**
Related: https://github.com/checkstyle/checkstyle/issues/13045

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
Not applying
https://github.com/checkstyle/checkstyle/actions/runs/6401606672

Maybe after merge it will be active.

**Review by romani — APPROVED** [review]
Let's try , ok to merge

**Comment by AayushSaini101** [comment]
> Let's try , ok to merge

If it is possible add hacktoberfest

File: .github/workflows/site.yml [hunk 1]
```diff
@@ -40,6 +40,12 @@ jobs:
       commit_sha: ${{ steps.branch.outputs.commit_sha }}
 
     steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
       - uses: khan/pull-request-comment-trigger@master
```
Comment (nrmancuso) [inline .github/workflows/site.yml:49]: Please update this action to use This comment is for the line below: please update  to use https://github.com/marketplace/actions/pull-request-comment-trigger-fork instead instead, node version is patched there.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Please revert current changes in this PR. We need to send a PR to update https://github.com/checkstyle/contribution/blob/f3239698c0cebdd81ad5f588d249a73ed0561fd8/comment-action/action.yml#L15 for the second warning in the issue description.


For the first warning:

File: .github/workflows/site.yml (hunk 1 above)
Comment (nrmancuso) [reply .github/workflows/site.yml:49]: We also need to update the other workflow that uses this action: https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20khan%2Fpull-request-comment-trigger&type=code

Comment (AayushSaini101) [reply .github/workflows/site.yml]: @nrmancuso Updated, For the 1st warning i am creating another PR

**Review by nrmancuso — APPROVED** [review]
Approved, @romani please take another look.

File: .github/workflows/site.yml (hunk 1 above)
Comment (nrmancuso) [reply .github/workflows/site.yml]: Leaving this unresolved for next reviewer to see.

**Review by romani — APPROVED** [review]
Let's try

**Comment by romani** [comment]
Please apply this patch to pass spelling https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=16855&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=38

**Comment by AayushSaini101** [comment]
> Please apply this patch to pass spelling https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=16855&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=38

Hi I added the keyword in whitelist list still it is failing

**Comment by romani** [comment]
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=16864&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=39

New word should be added at exact place. Just strictly follow patch command. Just copy it and apply in terminal

**Comment by romani** [comment]
And fix issue number in commit message, CI clearly pointing on fact that you issue is not approved.

**Pull request merged**